src/web/rustdoc.rs

```diff
@@ -2063,12 +2054,6 @@ mod test {
 
             let web = env.frontend();
 
-            assert!(!web
-                .get("/dummy-dash/0.1.0/dummy_dash/")
-                .send()?
-                .text()?
-                .contains("rel=\"canonical\""),);
```

Comment:
I would expect this to  still be an assert, just going the opposite way.
```suggestion
            assert!(web
                .get("/dummy-dash/0.1.0/dummy_dash/")
                .send()?
                .text()?
                .contains("rel=\"canonical\""),);
```